sql/log_event.cc

```diff
@@ -3674,11 +3674,27 @@ bool Query_log_event::write(Basic_ostream *ostream) {
     *start++ = thd->variables.default_table_encryption;
   }
 
+#ifdef WITH_WSREP
+  /*
+    Replicate Q_DDL_SKIP_REWRITE only if the session has
+    binlog_ddl_skip_rewrite set.
+  */
   if (thd && thd->variables.binlog_ddl_skip_rewrite) {
     *start++ = Q_DDL_SKIP_REWRITE;
     *start++ = thd->variables.binlog_ddl_skip_rewrite;
   }
 
+  /*
+    Replicate Q_WSREP_SKIP_READONLY_CHECKS only if it is a
+    replication applier thread.
```

Comment:
I suggest different wording to make it clear that we are talking about SQL replication applier thread, not WSREP replication applier

Reply:
Done